### PR TITLE
fix non import of java.util.Collections of branch-0.7

### PR DESCRIPTION
**Pull request by reminia**
fix branch-0.7 compiled failure of non import of Collections in NotebookServer.java

### CODE REVIEW

**Comment by reminia** [comment]
@Leemoonsoo

File: zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java [hunk 1]
```diff
@@ -19,14 +19,7 @@
...
 import java.net.UnknownHostException;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-import java.util.Queue;
-import java.util.Set;
+import java.util.*;
```
Comment (felixcheung) [inline zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java]: I think our style discourage importing `*`

Comment (Leemoonsoo) [inline zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java]: Thanks @reminia. 
Like @felixcheung mentioned, we're following [style](http://zeppelin.apache.org/contribution/contributions.html#code-convention) which discourages [whilecard imports](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports).

(style guide is not easy to find. i think we need to automate style check using stylecheck plugin)

 @reminia could you try resolve issue without using `*`?
Once it's updated, i'll merge this to branch-0.7 immediately as a hotfix.

Comment (reminia) [inline zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java]: @felixcheung @Leemoonsoo  thx for pointing out. Already append another commit.

**Comment by Leemoonsoo** [comment]
Merge to `branch-0.7` as a hotfix

**Comment by Leemoonsoo** [comment]
Just merged to `branch-0.7` but @asfgit does not close this one automatically.
@reminia Could you close this guy manually?

**Pull request closed without merging**